### ISSUE TITLE
Platform Game Assets Ultimate Wrong Hex Code Color 

### ISSUE THREAD

**Issue by Taiyri** [BayatGames/Support#19]
There appear to be some inconsistent colors between tiles that should connect but don't seem to. The bottom right portion of Spring-tilehill2-256 (the darkest portion of the dirt) is a different color than the darkest portion of Dirt-connectedtilehillleft2-256.

I have also noticed this problem with Dirt-lefttop-256. The second darkest portion of the dirt in the lower right appears to be a different color than the darkest portion of Dirt-midtop-256.

Dirt-lefttop-256's darkest color does match up with Dirt-midtop3-256 but then there's no connection to Dirt-connectedtilehillleft2-256.

I can provide pictures if necessary.

Also forgot to mention, these are in the 2.5d textures.

**Comment by aminbayat** [comment]
hi john! 

im designer of the platform game assets ultimate, many thanks for your time.
i checked the images and i didn't find any problem.
![Annotation 2020-01-15 105202](https://user-images.githubusercontent.com/31162334/72414645-f5aba180-3787-11ea-983d-9ece002d181c.jpg)
 it's very good if you send some pictures and dont worry about it we'll fix it.

**Comment by Taiyri** [comment]
I've attached some pictures below with arrows pointing to the colors that don't match up. I've also labeled the affected tiles for reference.

![2dplatformer issue](https://user-images.githubusercontent.com/24976286/72446244-ce010d00-3778-11ea-9a44-ce548a77f106.PNG)
![2dplatformer issue2](https://user-images.githubusercontent.com/24976286/72446246-d0636700-3778-11ea-98ee-7e7f6242e2f2.PNG)
![2dplatformer issue3](https://user-images.githubusercontent.com/24976286/72446251-d22d2a80-3778-11ea-9b61-27c23b33f8d6.PNG)

**Comment by aminbayat** [comment]
hello again, i designed a New TIle ( [New Dirt Tile.zip](https://github.com/BayatGames/Support/files/4066222/New.Dirt.Tile.zip) ) for that position ( thank you for noticed it) and i'll update the package.

![Annotation 2020-01-15 203044](https://user-images.githubusercontent.com/31162334/72454879-fae40d00-37d6-11ea-99ac-7f554efd46bb.jpg)

and about that colors, i copy your tile layout on my unity project but i didn't find any problem again.

![Annotation 2020-01-15 203314](https://user-images.githubusercontent.com/31162334/72455077-5adab380-37d7-11ea-9756-9646ba23d667.jpg)
![Annotation 2020-01-15 193525](https://user-images.githubusercontent.com/31162334/72455134-75ad2800-37d7-11ea-9b29-ba2839abc599.jpg)
![Annotation 2020-01-15 204858](https://user-images.githubusercontent.com/31162334/72455695-8a3df000-37d8-11ea-9120-4137537167dd.jpg)


maybe you have adjusted texture settings? or maybe you have some sort of plugin which adjusts the color of textures?

**Comment by Taiyri** [comment]
It appears to be an issue with Unity for some reason. I confirmed the same thing you did that the hex codes. I created a blank project and re-imported the pack and I'm still having the same issue. I'm hoping to get some help from the unity discord. I'm going to close this issue, as it's clearly a unity issue and not a problem with the asset.

Thank you for adding that extra tile though, that was very helpful and thank you for looking into it!
